go/worker/compute/txnscheduler/algorithm/batching/batching.go

```diff
@@ -44,29 +44,6 @@ type config struct {
 }
 
 func (s *batchingState) scheduleBatch(force bool) error {
-	// The simple batching algorithm only supports a single executor committee. Use
-	// with multiple committees will currently cause the rounds to fail as all other
-	// committees will be idle.
-	var committeeID *hash.Hash
-	func() {
-		// Guarding against EpochTransition() modifying current epoch.
-		s.RLock()
-		defer s.RUnlock()
-
-		// We cannot schedule anything until there is an epoch transition.
-		if s.epoch == nil {
```

Comment:
by removing this, i think the `s.epoch` field can be removed as well (and with that also the `EpochTransition` alg. interface method, and also the lock on the batching state)

Reply:
Good idea, removed.